inc/Config/AutoConf/ParamsUtil.pm

```diff
@@ -22,8 +22,11 @@ sub check_paramsutil_prerequisites
 {
     my $self = shift->_get_instance();
 
-    $self->{config}->{cc}                    or $self->check_prog_cc();
-    $self->check_produce_loadable_xs_build() or die "Can't produce loadable XS module";
+    if (!$self->check_pureperl_required())
+    {
+        $self->{config}->{cc}                    or $self->check_prog_cc();
+        $self->check_produce_loadable_xs_build() or die "Can't produce loadable XS module";
```

Comment:
I'd prefer to fallback to pureperl build here - it's a copy'n'paste error.
I can either modify your pull request or you force push an updated patch.

Reply:
I can modify the pull request, but I'm not sure how to fallback to pureperl. Is it just a matter of not dying if `$self->check_produce_loadable_xs_build()` fails?

Reply:
What do you think about https://github.com/perl5-utils/Params-Util/commits/nfg-rt133623 ?

Reply:
Looks good to me! I cherry-picked the new commit and pushed it up.

Reply:
Or you can just merge that nfg-rt133623 branch you made. Whatever works for you.

Reply:
I check it tomorrow morning and do a release then ;)

Reply:
Cool. Thanks so much!